### PR TITLE
[service-now] fix tests by changing ids

### PR DESCRIPTION
**Pull request by mmelko**
//test: `@servicenow`

### CODE REVIEW

File: ui-tests/src/test/resources/features/integrations/service-now.feature [hunk 1]
```diff
@@ -70,14 +70,15 @@ Feature: Integration - ServiceNow-amq/log
     Then check that position of connection to fill is "Finish"
 
     When select the "ServiceNow" connection
-    And select "Create Record" integration action
+    And select "Add Record" integration action
     And select "Add new Incident" from "table" dropdown
     And click on the "Next" button
+    Then  sleep for jenkins delay or "10" seconds
```
Comment (tplevko) [inline ui-tests/src/test/resources/features/integrations/service-now.feature]: two consequent white spaces

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/resources/features/integrations/service-now.feature [hunk 2]
```diff
@@ -31,8 +31,8 @@ Feature: Integration - ServiceNow-amq/log
     When select the "ServiceNow" connection
     And select "Retrieve Record" integration action
     And .*fill in values
```
Comment (avano) [inline ui-tests/src/test/resources/features/integrations/service-now.feature]: remove `.*`

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mmelko** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**